openshiftcontrolplane/v1/types.go

```diff
@@ -258,7 +258,10 @@ const (
 type OpenShiftControllerManagerConfig struct {
 	metav1.TypeMeta `json:",inline"`
 
-	KubeClientConfig configv1.KubeClientConfig `json:"kubeClientConfig"`
+	// KubeClientConfig is no longer being used.
```

Comment:
No longer? Or never? Was it ever actually implemented?

Reply:
And who typically writes to this object? Is this generally only written to by an operator?

Reply:
Yeah, it's used by OCMO or Hypershift. But neither accesses this field.

Hypershift only sets it statically in a YAML and it's been updated for the library-go way of config in https://github.com/openshift/hypershift/pull/6203

OCMO doesn't set the field, because OCM uses in-cluster config there and an explicit kubeconfig is not needed.